### PR TITLE
sort with swaps

### PR DESCRIPTION
**Pull request by Lannister1234**
@Kelvinson please do code review, thx

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            framework.setSquare(i, 0, Integer.toString(arr[i]));
+        }
+    }
+
+    @Override
+    public void onNewMove() {
+        System.out.println("Status of game: at method onNewMove()");
+    }
+
+    public boolean isMoveValid(int x, int y) {
```
Comment (Kelvinson) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: the parameters x, y are not used for the func

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            Integer j = Integer.parseInt(framework.getSquare(i+1, 0));
+            if (j < Integer.parseInt(framework.getSquare(i, 0))) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public String getGameOverMessage() {
```
Comment (Kelvinson) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:111]: cnt should be initialized before use

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 1 above)
Comment (Lannister1234) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: removed x, y from function parameters

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 2 above)
Comment (Lannister1234) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:111]: initialize cnt to 0 before use.

**Review by Kelvinson — COMMENTED** [review]
Great! Solve the 2nd and I can accept

**Pull request merged**